uc/uc.py

```diff
@@ -20,6 +20,7 @@ def marshalled(self, context):
         company_report = body[0]
         company_report.set('ns1:product', self.product_code)
         company_report.set('ns1:version', '2.1')
+        print(context.envelope.getChild('Body'))
```

Comment:
There is no need to create a handler, you just need an instance of a logger. It is not the the same because print statements are not actually logged and you cannot set a logging level. Here is what the Python documentation says about when to use logging: https://docs.python.org/2/howto/logging.html#when-to-use-logging